doc/explanation/containers_and_vms.md

```diff
@@ -3,22 +3,37 @@
 
 Incus provides support for two different types of {ref}`instances <expl-instances>`: *system containers* and *virtual machines*.
 
-When running a system container, Incus simulates a virtual version of a full operating system. To do this, it uses the functionality provided by the kernel running on the host system.
+When running a system container, Incus simulates a virtual version of a full operating system.  
+To do this, it uses the functionality provided by the kernel running on the host system.
```

Comment:
Line break isn't needed as it's not visible in the rendered document and so just makes us needlessly diverge from the matching text in the LXD doc.